src/tokenizer/lindera.rs

```diff
@@ -0,0 +1,96 @@
+use std::borrow::Cow;
+
+use lindera::tokenizer::Tokenizer as LinderaTokenizer;
+use lindera_core::core::viterbi::Mode;
+use once_cell::sync::Lazy;
+
+use crate::{Token, TokenKind};
+use crate::processors::ProcessedText;
+use super::{Tokenizer, TokenStream};
+
+
+#[derive(Debug, Default)]
+pub struct Lindera {
+    pub normal_mode: bool,
+    pub dict: &'static str,
```

Comment:
Jieba has a [default dictionary stored in a const](https://github.com/messense/jieba-rs/blob/7fe15e93cfc808a5ed5e2514c2eb61f7d57e4f01/src/lib.rs#L245-L256), could we do the same with lindera ?